client/reader/following/main.tsx

```diff
@@ -12,35 +10,14 @@ import SuggestionProvider from 'calypso/reader/search-stream/suggestion-provider
 import ReaderStream, { WIDE_DISPLAY_CUTOFF } from 'calypso/reader/stream';
 import ReaderListFollowedSites from 'calypso/reader/stream/reader-list-followed-sites';
 import Recent from '../recent';
+import { useSubscriptions } from './use-subscriptions';
```

Comment:
I can rename or relocate this if there are alternative suggestions.

Reply:
`useSitesSubscriptions` or `useFeedsSubscriptions` seems like a better name to me.

Reply:
Thanks! Changed to `useSiteSubscriptions` in https://github.com/Automattic/wp-calypso/pull/97030/commits/a9a533ca86b75b45601f7c12b83be29ce170e6e4